operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -272,6 +273,7 @@ spec:
 #      enabled: true
 #      namespace: ""
 #      service : ""
+#      port: 16686
```

Comment:
Oh man!! you guys screwed up my alphabetically sorting in here. I knew this would happen when I went on vacation :)

Reply:
Yeah, we broke as much as we could during your vacation!